classes/config/ConfigParser.php

```diff
@@ -3,30 +3,23 @@
 /**
  * @file classes/config/ConfigParser.php
  *
- * Copyright (c) 2014-2021 Simon Fraser University
- * Copyright (c) 2000-2021 John Willinsky
+ * Copyright (c) 2024 Simon Fraser University
```

Comment:
Not a big deal, but don't remove old dates when they're present -- just update the latest date if it's not current.